### PR TITLE
Paypal

### PR DESCRIPTION
**Pull request by Phildesro123**
Updated payment routing

### CODE REVIEW

File: server/models/household.js [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
-let Student = require('./student').schema;
+
+let Student = require('./student')
+
+
 let Schema = mongoose.Schema;
 
 let HouseholdSchema = new Schema({
-    students: [Student],
+    students: [Object],
```
Comment (SujeethJinesh) [inline server/models/household.js]: this change doesn't break anything?

**Comment by SujeethJinesh** [comment]
Undelete the README in client.

File: server/models/household.js (hunk 1 above)
Comment (Phildesro123) [reply server/models/household.js]: I believe it shouldn't

Comment (SujeethJinesh) [reply server/models/household.js]: Ok cool. did you check if it works on staging? The PR looks good to me otherwise.

Comment (Phildesro123) [reply server/models/household.js]: Yeah I'm doing that right now

Comment (SujeethJinesh) [reply server/models/household.js]: Sweet! Just check that it works, you might have to change the links to see the staging ones, so anything that's "bright-futures-academy-dev.herokuapp.com" --> "bright-futures-academy-staging.herokuapp.com". You can push this to a different branch and then deploy to staging so you don't mess up this PR

File: server/app.js [hunk 2]
```diff
@@ -12,11 +12,12 @@ var householdRouter = require('./routes/householdRouter');
...
+var paymentRouter = require('./routes/paymentRouter')
 var emailRouter = require('./routes/emailRouter')
 
 var app = express();
 
-// our dev server and client run on 2 different ports 
+// our dev server and client run on 2 different ports
 // so we need to whitelist the client address
 if (app.get('env') === 'development') {
   const whitelist = ['http://localhost:3000', 'https://bright-futures-academy-dev.herokuapp.com/']
```
Comment (SujeethJinesh) [inline server/app.js]: If you want to check staging, add "bright-futures-academy-staging.herokuapp.com" here

File: server/models/household.js (hunk 1 above)
Comment (spencerhyphengold) [reply server/models/household.js]: This change breaks something

Comment (SujeethJinesh) [reply server/models/household.js]: This is a pretty big PR, I think we should definitely go through it in person just so we can make sure we aren't breaking prod

File: server/testServer.js [hunk 3]
```diff
@@ -1,17 +1,18 @@
-
 //This is strictly for testing purposes
```
Comment (spencerhyphengold) [inline server/testServer.js]: Philippe, is there any reason to keep this file?

Comment (spencerhyphengold) [inline server/testServer.js]: Philippe, is there any reason to keep this file?

Comment (SujeethJinesh) [reply server/testServer.js]: Looks like something we'd use for testing, I'd be fine with keeping this file in.

File: client/src/components/PayPalConnect.js [hunk 4]
```diff
@@ -1,58 +1,109 @@
...
-        
+
         this.state = {
-            totalFee: 0.0, // TODO: replace these values with money values.
+            // TODO: get all of this information from Redux store
+            totalFee: 0.01,
             tax: 0.0,
-            paymentTotal: 0.0 // TODO: Make a state for checking to see if the user is connected to paypal already
+            paymentTotal: 0.01, 
+            householdId: '5ca780096bf2281dd453bdd8',
```
Comment (SujeethJinesh) [inline client/src/components/PayPalConnect.js]: We can actually get a real householdId and stuff now. But let's just keep this for a different PR.

File: server/models/household.js (hunk 1 above)
Comment (Phildesro123) [reply server/models/household.js]: Definitely, we should definitely check this out on Tuesday.

File: server/testServer.js (hunk 3 above)
Comment (spencerhyphengold) [reply server/testServer.js]: We should be able to run all of our tests in app.js in development mode, plus it'll be a single place that we know everything works instead of needing to keep the two up to date.

File: server/models/household.js (hunk 1 above)
Comment (Phildesro123) [reply server/models/household.js]: I believe that rebasing changed a few things, but right now it works on staging and has passed the tests.
